source/faq.txt

```diff
@@ -17,6 +17,21 @@ This page provides answers to some frequently asked questions we have
 encountered. If you have additional questions please contact MongoDB
 Support.
 
+Can I perform reads or writes to my destination cluster while ``mongosync`` is syncing?
+---------------------------------------------------------------------------------------
+
+You can perform reads when ``canCommit`` is ``true``. However, the data that you 
+read is :term:`eventually consistent <eventual consistency>`, meaning that you 
+may not always read the latest writes.
+
+You can write to any non-synced namespaces while sync is in progress. If you 
+write to  a namespace before issuing a :ref:`commit <c2c-api-commit>` and 
+waiting for ``canWrite: true``, the behavior is undefined. You can avoid this 
+undefined behavior by enabling :ref:`write blocking <c2c-dr-write-blocking>`.
```

Comment:
[Suggestion]

I think we can make this last sentence slightly more active.

To avoid this undefined behavior, you can enable `write blocking <link>`